mappings/net/minecraft/client/gui/DrawableHelper.mapping

```diff
@@ -55,32 +59,33 @@ CLASS net/minecraft/class_332 net/minecraft/client/gui/DrawableHelper
 		ARG 7 vStart
 		ARG 8 vEnd
 	METHOD method_25296 fillGradient (IIIIII)V
-		ARG 1 top
-		ARG 2 left
-		ARG 3 right
-		ARG 4 bottom
+		ARG 1 x1
+		ARG 2 y1
+		ARG 3 x2
+		ARG 4 y2
```

Comment:
parameter names not reverted here

Reply:
they were straight out wrong before.

Reply:
```suggestion
		ARG 1 xStart
		ARG 2 yStart
		ARG 3 xEnd
		ARG 4 yEnd
```

Reply:
@shedaniel are they interchangable here?

Reply:
oops, forgot to fix this.